geodetic_utils/include/geodetic_utils/geodetic_conv.h

```diff
@@ -0,0 +1,208 @@
+#ifndef GEODETIC_CONV_H_
+#define GEODETIC_CONV_H_
+
+#include "math.h"
+#include <Eigen/Dense>
+
+namespace geodetic_conv {
+// Geodetic system parameters
+static constexpr double kSemimajorAxis = 6378137;
+static constexpr double kSemiminorAxis = 6356752.3142;
+static constexpr double kFirstEccentricitySquared = 6.69437999014 * 0.001;
+static constexpr double kSecondEccentricitySquared = 6.73949674228 * 0.001;
+static constexpr double kFlattening = 1 / 298.257223563;
+
+class GeodeticConverter
+{
+ public:
+
+  // Default initialisation/copy constructor and assignment operator are OK.
+
+  ~GeodeticConverter()
+  {
+  }
+
+  void initialiseReference(const double lat, const double lon, const double altitude)
```

Comment:
no need for const prefacing these. It's copied over anyway, doesn't matter if the values are changed inside.